pkg/columns/columninfo.go

```diff
@@ -156,6 +156,23 @@ func (ci *Column[T]) fromTag(tag string) error {
 	return ci.parseTagInfo(tagInfo[1:])
 }
 
+func (ci *Column[T]) applyTemplate() error {
```

Comment:
Maybe also use this [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/fedee8c233071d29a29992b445eccb560e1b617a/pkg/columns/columns.go#L331)?